auth-api/src/auth_api/models/dataclass.py

```diff
@@ -45,3 +46,16 @@ class OrgSearch:  # pylint: disable=too-many-instance-attributes
     org_type: str
     page: int
     limit: int
+
+
+@dataclass
+class TaskSearch:
+    """Used for searching tasks."""
+
+    name: str
+    date_submitted: DateTime
```

Comment:
date should probably be a str, yyyy-mm-dd - keep in mind the database is UTC.. maybe better to look around for some other function for searching dates.

Reply:
we'll also have to chat with UX how they want this.. date picker?

Reply:
Try setting some defaults, that way they don't need to be included everytime? Can get away with str = ''

Reply:
I'll ping janis about if we want a date picker or sorting and what we want the components to look like